scripts/settings.yaml

```diff
@@ -0,0 +1,511 @@
+---
+- group: base_station_mode
+  name: enable
+  type: boolean
+  units:
+  default value: 'false'
+  enumerated possible values: true,false
+  Description: Operate in base station mode
```

Comment:
Perhaps this description should be more specific e.g. "Output surveyed base station location"?

Perhaps we should consider renaming the setting too, in order to make it clear that in general its not necessary to set this to use Piksi as a base station.